config-overrides.js

```diff
@@ -0,0 +1,105 @@
+const webpack = require('webpack');
+const LavaMoatPlugin = require('@lavamoat/webpack')
+const fs = require('fs');
+const path = require('path');
+const stdLibBrowser = require('node-stdlib-browser');
+
+module.exports = function override(config, env) {
+  // Enable source maps for better debugging
+  config.devtool = 'source-map';
+
+  config.optimization = {
+    ...config.optimization,
+    minimize: false,
+    concatenateModules: false
+  };
+
+  // Configure module resolution
+  config.resolve = {
+    ...config.resolve,
+    fallback: {
+      buffer: require.resolve('buffer/'),
+      assert: stdLibBrowser.assert,
+      crypto: stdLibBrowser.crypto,
+      path: stdLibBrowser.path,
+      process: stdLibBrowser.process,
+      stream: stdLibBrowser.stream,
+      os: stdLibBrowser.os,
+      vm: false,
```

Comment:
What's this?

Reply:
It means we're not polyfilling it, `false` is the default, but react-scripts gives a warning if we don't do it:

```
BREAKING CHANGE: webpack < 5 used to include polyfills for node.js core modules by default.
This is no longer the case. Verify if you need this module and configure a polyfill for it.

If you want to include a polyfill, you need to:
        - add a fallback 'resolve.fallback: { "vm": require.resolve("vm-browserify") }'
        - install 'vm-browserify'
If you don't want to include a polyfill, you can use an empty module like this:
        resolve.fallback: { "vm": false }
```